docs/_future/app_manifests.md

```diff
@@ -0,0 +1,106 @@
+  name: 'take-your-time',
+  displayName: 'Take Your Time',
+  // A description of your app that will help users decide whether to use it.
+  description: 'Request and take time off.',
+  longDescription: 'Take your time off by using this application to request and take time off from your manager. Launch the workflow, put in your manager, requested PTO start and end date, and receive updates on your PTO request!',
+  botScopes: ['chat:write'],
+  tokenManagementEnabled: true,
+  socketModeEnabled: true,
+  // A list of all workflows your app will use.
+  workflows: [TimeOffWorkflow],
+  features: {
+    appHome: {
+      homeTabEnabled: true,
+      messagesTabEnabled: true,
+      messagesTabReadOnlyEnabled: true,
+    },
+  },
+  settings: {
+    interactivity: {
+      is_enabled: true,
+    },
+  },
+  tokenRotationEnabled: false,
+});
+```
+---
+
+### Manifest properties {#manifest-properties}
+
+The properties in the Manifest are:
+
+|Property|Type|Description|
+|---|---|---|
+| `name` | String | This is the internal name for your app. It can contain spaces (e.g., "My App") |
+| `description` |String| A short sentence describing your application. A description of your app that will help users decide whether to use it |
+| `displayName` | String | (Optional) Allows a custom name for the app to be displayed that's different from the `name` |
+| `longDescription` | String | (Optional) A more detailed description of your application |
+| `icon` | String | A relative path to an image asset to use for the app's icon. Your app's profile picture that will appear in the Slack client |
+| `backgroundColor` | String | (Optional) A six digit combination of numbers and letters (the hexadecimal color code) that make up the color of your app background e.g., "#000000" is the color black |
+| `botScopes` | Array<string> | A list of [scopes](/scopes), or permissions, the app's Functions require |
+| `functions` | Array | (Optional) A list of all Functions your app will use |
+| `workflows` | Array | (Optional) A list of all Workflows your app will use |
+| `outgoingDomains` | Array<string> | (Optional) If your app communicates to any external domains, list them here. Note that the outgoing domains are only restricted if the workspace has Admin approved apps on e.g., myapp.tld |
+| `events` | Array | (Optional) A list of all Event structures that the app is expecting to be passed via [Message Metadata](/metadata/using) |
+| `types` | Array | (Optional) A list of all [custom types](https://api.slack.com/future/types/custom) your app will use | 
+| `datastores` | Array | (Optional) A list of all [Datastores](https://api.slack.com/future/datastores) your app will use |
+| `features` | Object | (Optional)  A configuration object of your app features |
+
+You will come back to the Manifest every time you create a new workflow, since all workflows for your app need to be added to the Manifest in order to use them.
+
+---
+
+### The Messages tab {#messages-tab}
```

Comment:
@hello-ashleyintech - This isn't true for `runOnSlack:false` apps, so I think we can remove this section. The `Manifest` does not make assumptions when it can't be assumed that it's not a run on slack app, so it does not default.